src/lua/compat.c

```diff
@@ -5,8 +5,42 @@
  */
 
 #include <lua/compat.h>
+#include "serializer.h"
+
+static int
+lbox_msgpuck_escape_forward_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool esc_slash = lua_toboolean(L, -1);
+
+	/*
+	 * The table is changed outright, because runtime check (like one in
+	 * json serializer) is slower, more comolicated and is not required here
+	 * as there is no need to have different behavior for different mp
+	 * instances, unlike in json.
+	 */
+	mp_char2escape['/'] = esc_slash ? "\\/" : NULL;
+
+	return 0;
+}
+
+static int
+lbox_json_escape_forward_slash_toggle(struct lua_State *L)
+{
+	assert(lua_isboolean(L, -1) && "boolean argument expected");
+	bool esc_slash = lua_toboolean(L, -1);
+
+	assert(serializer_set_option_default("encode_escape_forward_slash",
+					     esc_slash) == 0);
+
+	return 0;
+}
 
 static const struct luaL_Reg internal_compat[] = {
+	{"msgpuck_escape_forward_slash_toggle",
+		lbox_msgpuck_escape_forward_slash_toggle},
```

Comment:
This is a rather strange indent. I think it should something like this:

```Diff
diff --git a/src/lua/compat.c b/src/lua/compat.c
index 0c3877f98..22d46b1ea 100644
--- a/src/lua/compat.c
+++ b/src/lua/compat.c
@@ -38,9 +38,9 @@ lbox_json_escape_forward_slash_toggle(struct lua_State *L)
 
 static const struct luaL_Reg internal_compat[] = {
        {"msgpuck_escape_forward_slash_toggle",
-               lbox_msgpuck_escape_forward_slash_toggle},
+        lbox_msgpuck_escape_forward_slash_toggle},
        {"json_escape_forward_slash_toggle",
-               lbox_json_escape_forward_slash_toggle},
+        lbox_json_escape_forward_slash_toggle},
        {NULL, NULL},
 };
 
```

Reply:
Fixed.